tests/contrib/django/test_django_appsec.py

```diff
@@ -380,3 +380,74 @@ def test_django_weak_hash(client, test_spans, tracer):
         vulnerability = json.loads(root_span.get_tag(IAST_JSON))["vulnerabilities"][0]
         assert vulnerability["location"]["path"].endswith("tests/contrib/django/views.py")
         assert vulnerability["evidence"]["value"] == "md5"
+
+
+def test_request_ipblock_nomatch_200(client, tracer):
+    with override_global_config(dict(_appsec_enabled=True)), override_env(dict(DD_APPSEC_RULES=RULES_GOOD_PATH)):
+        tracer._appsec_enabled = True
+        # Hack: need to pass an argument to configure so that the processors are recreated
+        tracer.configure(api_version="v0.4")
+
+        result = client.get("/?a=1&b&c=d", HTTP_X_REAL_IP="8.8.8.8")
+        assert result.status_code == 200
+        assert result.content == b"Hello, test app."
+
+
+def test_request_ipblock_match_403(client, test_spans, tracer):
+    with override_global_config(dict(_appsec_enabled=True)), override_env(dict(DD_APPSEC_RULES=RULES_GOOD_PATH)):
+
+        tracer._appsec_enabled = True
+        # # Hack: need to pass an argument to configure so that the processors are recreated
+        tracer.configure(api_version="v0.4")
+
+        result = client.get("/?a=1&b&c=d", HTTP_X_REAL_IP="8.8.4.4", HTTP_ACCEPT="text/html")
```

Comment:
this is another place where a constant might just make these tests easier to read, it is easy to miss that `8.8.8.8` and `8.8.4.4` are meant to behave differently.

Reply:
Done!